services/src/main/groovy/org/jfrog/artifactory/client/impl/ArtifactoryImpl.java

```diff
@@ -272,6 +272,29 @@ protected Boolean head(String path) throws IOException {
         return Util.responseToObject(httpResponse, object, interfaceObject);
     }
 
+    public <T> T patch(String path, org.apache.http.entity.ContentType contentType, String content, Map<String, String>
```

Comment:
Would it be possible to add `PATCH` to `org.jfrog.artifactory.client.ArtifactoryRequest.Method` and handle it in  `org.jfrog.artifactory.client.impl.ArtifactoryImpl#restCall` as well?